src/encoded/static/components/user.js

```diff
@@ -206,6 +206,11 @@ const User = (props) => {
         { id: 'Users' },
     ];
     const crumbsReleased = (context.status === 'released');
+    const hasSubmitterForRights = context.submits_for && context.submits_for.length > 0;
+    const isAdmin = context.groups && context.groups.includes('admin');
+    const hasViewGroupRights = context.viewing_groups && context.viewing_groups.length > 0;
+    const hasAccessKeyRights = (hasSubmitterForRights || isAdmin || hasViewGroupRights) && context.access_keys;
```

Comment:
Does `context.access_keys` have a possibility of not existing? I realize the existing code does this same test, but I’m wondering if it’s ever not truthy?

Reply:
It should always exist and the code should blow up if it does not. But never say never and to make things clearer, so I will check for existence.

Reply:
You’re already checking if it exists. I was wondering if you need to.